src/jit/importer.cpp

```diff
@@ -10203,6 +10296,7 @@ void Compiler::impImportBlockCode(BasicBlock* block)
         var_types       lclTyp, ovflType = TYP_UNKNOWN;
         GenTreePtr      op1           = DUMMY_INIT(NULL);
         GenTreePtr      op2           = DUMMY_INIT(NULL);
+        GenTree*        optTree       = nullptr;
```

Comment:
I would prefer to create new scopes under `CEE_ISINST` and `CASTCLASS:` and declare `optTree` there.

Reply:
Sure, will do.